draft-foudil-securitytxt.md

```diff
@@ -55,12 +55,18 @@ In this document, the key words "MUST", "MUST NOT", "REQUIRED",
 "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY",
 and "OPTIONAL" are to be interpreted as described in {{!RFC2119}}.
 
+# Note to Readers
+
+> **Note to the RFC Editor:**  Please remove this section prior
+> to publication.
+
+Development of this draft takes place on Github at: https://github.com/securitytxt/security-txt
+
 # The Specification
 
-security.txt is a text file that should be located under the
+security.txt is a text file that SHOULD be located under the
 /.well-known/ path ("/.well-known/security.txt") {{!RFC5785}} for web
-properties. For file systems and version control repositories a .security.txt
-file should be placed in the root directory. This text file contains 4 directives
+properties. A security.txt file located under the website's top-level directory can either redirect to the security.txt file located under the /.well-known/ path or be used as a fall back. For web-based services, the instructions must be accessible via the Hypertext Transfer Protocol {{!RFC1945}} as a resource of Internet Media Type "text/plain;charset=utf8" {{!RFC1590}}. For file systems and version control repositories a .security.txt file SHOULD be placed in the root directory. This text file contains 4 directives
```

Comment:
The reference to RFC1590 should probably be changed to point to section 4.1.3 of RFC2046 as per the rest of the draft, or maybe the media type registration.

For the charset, we may want to say something like "should be served with the default charset parameter set to "utf-8"